robottelo/cli/contentview.py

```diff
@@ -158,3 +158,10 @@ def remove_from_environment(cls, options=None):
         """Remove content-view from an enviornment"""
         cls.command_sub = 'remove-from-environment'
         return cls.execute(cls._construct_command(options))
+
+    @classmethod
+    def remove(cls, options=None):
+        """Remove versions and/or environments from a content view and
+        reassign content hosts and keys"""
```

Comment:
14 doc strings (actually **all**) before has another syntax. I prefer to fix all of the than put my unique doc string according to your comment. What do you think?